app/scripts/views/openid/login.js

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'cocktail',
+  'views/base',
+  'stache!templates/openid/login',
+  'lib/session'
+],
+function (Cocktail, BaseView, Template, Session) {
+  'use strict';
+
+  var View = BaseView.extend({
+    template: Template,
+    className: 'openid-sign-in',
+
+    initialize: function (options) {
+      BaseView.prototype.initialize.call(this, options);
+      options = options || {};
+
+      Session.clear();
+      this.user.clearSignedInAccount();
+    },
+
+    beforeRender: function () {
+      var self = this;
+      var loginUrl = '/account/openid/login' + window.location.search;
+      return this.fxaClient._client.request.send(loginUrl, 'GET')
```

Comment:
If we end up keeping it, it should move to fxa-js-client since its part of the auth-server api. I did it this way as an interim step.